torch/_inductor/ir.py

```diff
@@ -4678,9 +4686,13 @@ def codegen(self, wrapper):
                     schema = kernel._schema
 
                     self.args_default_value = [
-                        {"type": x.real_type, "value": x.default_value}
+                        {
+                            "name": x.name,
+                            "type": x.real_type,
+                            "value": x.default_value,
+                            "kwarg_only": x.kwarg_only,
+                        }
                         for x in schema.arguments
-                        if not x.kwarg_only
```

Comment:
I guess you had to drop this `if not x.kwarg_only` to make it work?

Reply:
No, I don't think it should be removed.

Reply:
On second thought, there's no need to remove it. I've updated the code.